scripts/install/mylar.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Mylar Installer for Swizzin
+# Author: Public920
+
+if [[ -f /tmp/.install.lock ]]; then
+    log="/root/logs/install.log"
+else
+    log="/root/logs/swizzin.log"
+fi
+
+user=$(cut -d: -f1 < /root/.master.info)
+codename=$(lsb_release -cs)
+. /etc/swizzin/sources/functions/pyenv
+
+if [[ $codename =~ ("buster"|"focal") ]]; then
+    LIST='git python3-dev virtualenv python3-virtualenv python3-pip'
+else
+    LIST='git build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev curl libbz2-dev'
+fi
+
+apt-get -y update >>"$log" 2>&1
+for depend in $LIST; do
+    apt-get -qq -y install $depend >>"$log" 2>&1 || { echo "ERROR: APT-GET could not install a required package: ${depend}. That's probably not good..."; }
+done
+
+if [[ ! $codename =~ ("buster"|"focal") ]]; then
+    cd /tmp
+    curl -O https://www.python.org/ftp/python/3.8.1/Python-3.8.1.tar.xz
+    tar -xf Python-3.8.1.tar.xz
+    cd Python-3.8.1
+    make
+    make install
+
+    python3_getpip
+fi
+
+python3_venv ${user} mylar
```

Comment:
Please take a look at how some of the other apps are being installed using the already defined and shared python venving functions. If there is something here that is re-usable, please make a function for it as well.

Reply:
i.e. From my limited knowledge of how python apps are being handled right now, we only venv things when python2 is needed and we let the python3 apps be handled normally unless something specific is needed.